conjur/logout/logout_controller.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+"""
+LogoutController module
+
+This module is the controller that facilitates all logout actions
+required to successfully logout
+"""
+
+# Builtins
+import logging
+import sys
+
+# Internals
+from conjur.utils import Utils
+
+# pylint: disable=too-few-public-methods
+class LogoutController:
+    """
+    LogoutController
+
+    This class represents the Presentation Layer for the LOGOUT command
+    """
+    logout_logic = None
+    ssl_verify = None
+
+    def __init__(self, ssl_verify, logout_logic):
+        self.ssl_verify = ssl_verify
+        if self.ssl_verify is False:
+            Utils.insecure_message()
+
+        self.logout_logic = logout_logic
+
+    def remove_credentials(self):
+        """
+        Method for removing credentials from user machine
+        """
+        logging.debug("Attempting to log out of Conjur")
+        try:
+            self.logout_logic.remove()
+            logging.debug("Logout successful")
+            sys.stdout.write("Logged out from Conjur\n")
+        except Exception as error:
+            # pylint: disable=raise-missing-from
+            raise Exception(f"Failed to logout. {error}")
```

Comment:
Suggestion:
```suggestion
            raise Exception(f"Failed to logout. Reason: {error}")
```

Reply:
Then it will be `Reason: Please login` which is no bueno

Reply:
You shouldn't catch and your own thrown exception in the same try/ecxept.
raise the exception outside, if you need to.
Still, IMO, you don't need to, as log out finished successfully if user is not logged in, and no need to fail it.
Also, `Please log in` is not an error, exception should contain the failure info, so `Reason: ` suggestion apply.

Reply:
maybe change the `pelase_login` into` "user not logged in"`